rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -223,7 +262,12 @@ def _decode_repay(
 
     def _decode_lending_pool_events(self, context: DecoderContext) -> DecodingOutput:
         """Decodes AAVE V2 Lending Pool events"""
-        if context.tx_log.topics[0] not in (ENABLE_COLLATERAL, DISABLE_COLLATERAL, DEPOSIT, WITHDRAW, BORROW, REPAY):  # noqa: E501
+        if context.tx_log.topics[0] not in (LIQUIDATION_CALL, ENABLE_COLLATERAL, DISABLE_COLLATERAL, DEPOSIT, WITHDRAW, BORROW, REPAY):  # noqa: E501
```

Comment:
nitpick: it could be split in multiple lines since there are already quite a bit of events types